.github/workflows/flutter.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         target: ['ios', 'android', 'web']
-        channel: ['stable', 'beta']
+        channel: ['stable']
```

Comment:
instead of removing the `channel`, lets exclude the channel `beta` for iOS only, see the `exclude` on this file

Reply:
maybe lets keep like this as its broken for Android too but lets write a comment why we've commented out the beta channel and link to the GH issue